main.js

```diff
@@ -5,7 +5,9 @@ const isDev = require('electron-is-dev')
 
 // Let electron reloads by itself when webpack watches changes in ./app/
 if (isDev) {
-  require('electron-reload')(__dirname)
+  require('electron-reload')(__dirname, {
+    electron: require('${__dirname}/../../node_modules/electron')
```

Comment:
Please revert this. The idea is to reload if any file under the Deer folder changes. Since it is dev-only we are ok with having the warning.